crates/goose-llm/src/types/json_value_ffi.rs

```diff
@@ -1,84 +1,18 @@
-use serde::{Deserialize, Serialize};
 use serde_json::Value;
 
 // `serde_json::Value` gets converted to a `String` to pass across the FFI.
 // https://github.com/mozilla/uniffi-rs/blob/main/docs/manual/src/types/custom_types.md?plain=1
 // https://github.com/mozilla/uniffi-rs/blob/c7f6caa3d1bf20f934346cefd8e82b5093f0dc6f/examples/custom-types/src/lib.rs#L63-L69
 
-#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
-pub struct JsonValueFfi(Value);
-
-impl From<JsonValueFfi> for Value {
-    fn from(val: JsonValueFfi) -> Self {
-        val.0
-    }
-}
-
-impl From<Value> for JsonValueFfi {
-    fn from(val: Value) -> Self {
-        JsonValueFfi(val)
-    }
-}
-
-uniffi::custom_type!(JsonValueFfi, String, {
+uniffi::custom_type!(Value, String, {
```

Comment:
updated based on the recommendation here: https://github.com/mozilla/uniffi-rs/issues/2533#issuecomment-2868051650